### ISSUE TITLE
Update runtime settings documentation with examples

### ISSUE THREAD

**Issue by burtlo** [habitat-sh/habitat#3167]
The runtime settings talks about the `sys` and `pkg` values in an abstract way that would benefit from having an example file structure to compare it against to have a better visual understanding of the location of the package paths and package service paths. Additional examples like https://www.habitat.sh/tutorials/sample-app/mac/add-health-check-hook/ would help demonstrate some of the places where this content could be used within a hook or configuration file. 

A reference to hooks (https://www.habitat.sh/docs/reference/hooks/) and package configuration (https://www.habitat.sh/docs/create-packages-configure/) would be useful.

**Comment by tashimi** [comment]
This is kind of a duplicate of what I wanted out of #2881, which you could basically turn into a blog once the work described in this issue is done.

**Comment by nellshamrell** [comment]
Closing in favor of #2881 - ty so much @burtlo!
